go/systests/stellar_test.go

```diff
@@ -61,9 +71,125 @@ func TestStellarNoteRoundtripAndResets(t *testing.T) {
+	require.NoError(t, err)
+	gift(t, res[0].AccountID)
+
+	t.Logf("alice sends a first relay payment to bob P1")
+	attachIdentifyUI(t, alice.tc.G, newSimpleIdentifyUI())
+	cmd := client.CmdWalletSend{
+		Contextified: libkb.NewContextified(alice.tc.G),
+		Recipient:    bob.username,
+		Amount:       "50",
+	}
+	require.NoError(t, cmd.Run())
+
+	t.Logf("alice sends a second relay payment to bob P2")
+	cmd = client.CmdWalletSend{
+		Contextified: libkb.NewContextified(alice.tc.G),
+		Recipient:    bob.username,
+		Amount:       "30",
+	}
+	require.NoError(t, cmd.Run())
+
+	t.Logf("get the impteam seqno to wait on later")
+	team, _, _, err := teams.LookupImplicitTeam(context.Background(), alice.tc.G, alice.username+","+bob.username, false)
+	require.NoError(t, err)
+	nextSeqno := team.NextSeqno()
+
+	t.Logf("bob gets a PUK and wallet")
+	bob.perUserKeyUpgrade()
+	bob.tc.G.GetStellar().CreateWalletSoft(context.Background())
+
+	t.Logf("wait for alice to add bob to their impteam")
+	alice.pollForTeamSeqnoLinkWithLoadArgs(keybase1.LoadTeamArg{ID: team.ID}, nextSeqno)
+
+	pollFor(t, "claims to complete", 10*time.Second, bob.tc.G, func(i int) bool {
+		res, err = bob.stellarClient.GetWalletAccountsLocal(context.Background(), 0)
+		require.NoError(t, err)
+		t.Logf("poll-1-%v: %v", i, res[0].BalanceDescription)
+		if res[0].BalanceDescription == "0 XLM" {
+			return false
+		}
+		if res[0].BalanceDescription == "49.9999800 XLM" {
+			t.Logf("poll-1-%v: received T1 but not T2", i)
+			return false
+		}
+		if res[0].BalanceDescription == "29.9999800 XLM" {
+			t.Logf("poll-1-%v: received T2 but not T1", i)
+			return false
+		}
+		t.Logf("poll-1-%v: received both payments", i)
+		require.Equal(t, "79.9999700 XLM", res[0].BalanceDescription)
+		return true
+	})
+
+	t.Logf("--------------------")
+	t.Logf("Part 2: Alice sends a relay payment to bob who now already has a wallet")
+	cmd = client.CmdWalletSend{
+		Contextified: libkb.NewContextified(alice.tc.G),
+		Recipient:    bob.username,
+		Amount:       "10",
+		ForceRelay:   true,
+	}
+	require.NoError(t, cmd.Run())
+
+	pollFor(t, "final claim to complete", 10*time.Second, bob.tc.G, func(i int) bool {
+		res, err = bob.stellarClient.GetWalletAccountsLocal(context.Background(), 0)
+		require.NoError(t, err)
+		t.Logf("poll-2-%v: %v", i, res[0].BalanceDescription)
+		if res[0].BalanceDescription == "79.9999700 XLM" {
+			return false
+		}
+		t.Logf("poll-1-%v: received final payment", i)
+		require.Equal(t, "89.9999600 XLM", res[0].BalanceDescription)
+		return true
+	})
+
+}
+
 func sampleNote() stellar1.NoteContents {
 	return stellar1.NoteContents{
 		Note:      "wizbang",
 		StellarID: stellar1.TransactionID("6653fc2fdbc42ad51ccbe77ee0a3c29e258a5513c62fdc532cbfff91ab101abf"),
 	}
 }
+
+// Friendbot sends someone XLM
+func gift(t testing.TB, accountID stellar1.AccountID) {
+	t.Logf("gift -> %v", accountID)
+	url := stellarnet.Client().URL + "/friendbot?addr=" + accountID.String()
```

Comment:
We should change to the new URL:

    https://friendbot.stellar.org/?addr=whatever

even though the one you have here is redirecting.

I was told to use `friendbot.stellar.org` by sdf.